app/code/Magento/Sales/Controller/AbstractController/PrintInvoice.php

```diff
@@ -17,64 +25,95 @@ abstract class PrintInvoice extends \Magento\Framework\App\Action\Action
     protected $orderAuthorization;
 
     /**
-     * @var \Magento\Framework\Registry
+     * @var Registry
      */
-    protected $_coreRegistry;
+    protected $coreRegistry;
 
     /**
      * @var PageFactory
      */
     protected $resultPageFactory;
 
+    /**
+     * @var InvoiceRepositoryInterface
+     */
+    protected $invoiceRepository;
+
+    /**
+     * @var Session
+     */
+    protected $session;
+
+    /**
+     * @var Order
+     */
+    protected $order;
+
     /**
      * @param Context $context
      * @param OrderViewAuthorizationInterface $orderAuthorization
-     * @param \Magento\Framework\Registry $registry
+     * @param Registry $registry
      * @param PageFactory $resultPageFactory
      */
     public function __construct(
-        Context $context,
+        Context                         $context,
         OrderViewAuthorizationInterface $orderAuthorization,
-        \Magento\Framework\Registry $registry,
-        PageFactory $resultPageFactory
+        Registry                        $registry,
+        PageFactory                     $resultPageFactory,
+        InvoiceRepositoryInterface      $invoiceRepository,
+        Session                         $session,
+        Order                           $order
```

Comment:
Can the additional indentation be removed please? If more dependancies are added this could result in all the indentation for the dependancies having to be re-lined up.